<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roleForm</class>
 <widget class="QWidget" name="roleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>511</width>
    <height>392</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>511</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txt_Name"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_Del">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addSkill">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Skill</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Roles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Add">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_rmSkill">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Skill</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_Mskills">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>141</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>160</width>
     <height>126</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>HP</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="num_ATKx">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="num_HPc">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="num_DEFc">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Defense</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="num_DEFx">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="num_ATKc">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="num_MPc">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Attack</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="num_MPx">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>MP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="num_HPx">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lst_Mskills</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>btn_Del</zorder>
  <zorder>btn_addSkill</zorder>
  <zorder>lst_Roles</zorder>
  <zorder>btn_Add</zorder>
  <zorder>btn_rmSkill</zorder>
  <zorder>gridLayoutWidget_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
